packages/block-library/src/media-text/index.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Server-side rendering of the `core/media-text` block.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Renders the `core/media-text` block on server.
+ *
+ * @param array  $attributes The block attributes.
+ * @param string $content    The block rendered content.
+ *
+ * @return string Returns the Media & Text block markup, if useFeaturedImage is true.
+ */
+function render_block_core_media_text( $attributes, $content ) {
+	if ( false === $attributes['useFeaturedImage'] ) {
+		return $content;
+	}
+
+	if ( in_the_loop() ) {
+		update_post_thumbnail_cache();
+	}
+
+	$current_featured_image = get_the_post_thumbnail_url();
+	if ( ! $current_featured_image ) {
+		return $content;
+	}
+
+	$processor = new WP_HTML_Tag_Processor( $content );
+	if ( isset( $attributes['imageFill'] ) && $attributes['imageFill'] ) {
+		if ( isset( $attributes['focalPoint'] ) ) {
+			$position = round( $attributes['focalPoint']['x'] * 100 ) . '% ' . round( $attributes['focalPoint']['y'] * 100 ) . '%';
+		} else {
+			$position = '50% 50%';
+		}
```

Comment:
```suggestion
		$position = '50% 50%';
		if ( isset( $attributes['focalPoint'] ) ) {
			$position = round( $attributes['focalPoint']['x'] * 100 ) . '% ' . round( $attributes['focalPoint']['y'] * 100 ) . '%';
		}
```

We can reduce the amount of code by just one line. Additionally, it would match the style of this passage.

https://github.com/WordPress/gutenberg/pull/51491/files#diff-e65105b1f1e325d53864a6ee67246b9f13e65d6fc359ede052dd925c647843adR41-R44